docs/source/guides/ethereum_node.rst

```diff
@@ -0,0 +1,271 @@
+        > personal.newAccount();
+        > eth.accounts
+        ["0x287a817426dd1ae78ea23e9918e2273b6733a43d"]
+
+* MacOS:
+
+    .. code:: bash
+
+        $ geth attach /Users/<username>/Library/Ethereum/geth.ipc
+        > personal.newAccount();
+        > eth.accounts
+        ["0x287a817426dd1ae78ea23e9918e2273b6733a43d"]
+
+Where ``0x287a817426dd1ae78ea23e9918e2273b6733a43d`` is your newly created
+account address and ``<username>`` is your user.
+
+.. note::
+
+    The Geth console **does not return EIP-55 compliant checksum addresses**, and instead will output
+    the *lowercase* version of the address.  Since Nucypher requires EIP-55 checksum addresses, you will need
+    to convert your address to checksum format:
+
+    .. code:: javascript
+
+       > web3.toChecksumAddress(eth.accounts[0])
+       "0x287A817426DD1AE78ea23e9918e2273b6733a43D"
+
+
+Run Geth with Docker
+********************
+
+Run a local geth using volume bindings:
+
+.. code:: bash
+
+    docker run -it -p 30303:30303 -v ~/.ethereum:/root/.ethereum ethereum/client-go
+
+For alternate geth configuration via docker see:
+`Geth Docker Documentation <https://geth.ethereum.org/docs/install-and-build/installing-geth#run-inside-docker-container>`_.
+
+
+
+Remote Ethereum Node
+~~~~~~~~~~~~~~~~~~~~
+
+Nucypher supports remote ethereum providers such as Alchemy, Infura, Public Remote Node, but an external transaction
+signing client is needed separate from the broadcasting node.
+
+
+External Transaction Signing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In conjunction with an ethereum provider, an external transaction signer can be specified and operated
+independently of the provider/broadcaster. This separation allows pre-signed transactions to be sent to an
+external (possibly remote) ethereum node and is particularly desirable when interacting with an untrusted
+ethereum node.
+
+For example, external signers can be used with:
+
+- Infura/Alchemy/Etc.
+- Local ethereum node
+- Remote ethereum node
+
+Two external signers are currently supported:
+
+#. `Signing with Clef`_
+#. `Signing with Local Keystore`_
+
+
+.. important::
+
+    External signing support is an experimental feature and under active development.
+
+
+.. _signing-with-clef:
+
+Signing with Clef
+*****************
+
+Clef enables applications to connect to an ethereum node and send locally signed
+transactions to be broadcasted. More
+information about Clef can be found `here <https://github.com/ethereum/go-ethereum/tree/master/cmd/clef>`_. Clef can
+use hardware wallets (ledger and trezor) over USB, or geth formatted private keys by specifying the keystore
+directory path.
+
+
+Clef Setup
+++++++++++
+
+We'll quickly walk through setup steps below, but additional in-depth documentation on Clef can
+be found in the `source repository <https://github.com/ethereum/go-ethereum/tree/master/cmd/clef>`_.
+
+Clef is typically installed alongside geth.  If you already have geth installed on your system you
```

Comment:
Last time I tried, with geth's last version (1.9.14), default geth installation doesn't include clef; I had to install the "Geth & Tools" version.

Reply:
Just tried installing get on Ubuntu (docker) and clef was installed by default:
```
$add-apt-repository -y ppa:ethereum/ethereum
$apt-get update
$apt-get install ethereum
```

version `1.9.14-stable-6d74d1e5` and `clef` was installed.